src/Particular.CodeRules/tools/install.ps1

```diff
@@ -0,0 +1,6 @@
+﻿param($installPath, $toolsPath, $package, $project)
```

Comment:
@SimonCropp ok, I did some checks, because I wasn't sure myself... it seems that installing analyzers adds them into the csproj as e.g.:

```xml
  <ItemGroup>
    <Analyzer Include="packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\Newtonsoft.Json.dll" />
    <Analyzer Include="packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll" />
    <Analyzer Include="packages\StyleCop.Analyzers.1.0.0\analyzers\dotnet\cs\StyleCop.Analyzers.dll" />
  </ItemGroup>
```

So after package restore they will work, because references remain. I guess `install.ps1` is just needed for the initial install.